crates/matrix-sdk-base/src/client.rs

```diff
@@ -143,7 +143,7 @@ impl BaseClient {
     ///   previous login call.
     pub fn with_store_config(config: StoreConfig) -> Self {
         let (room_info_notable_update_sender, _room_info_notable_update_receiver) =
-            broadcast::channel(100);
+            broadcast::channel((u16::MAX - 1) as usize);
```

Comment:
nit: `u16::MAX` is already 2**16 - 1, so no need to subtract one :nerd_face:

Reply:
Done.